tests/integrations/test_db_integrations_sql.py

```diff
@@ -192,9 +210,26 @@ def test_error_stack(self):
         span = self._get_sql_span_for_request(self.requests[self.db_service]["select_error"])
         assert span["meta"]["error.stack"].strip()
 
+    def _get_sql_span_for_request(self, weblog_request):
+        """To be implemented by subclasses"""
+        return {}
+
+
+@scenarios.integrations
```

Comment:
for the code clarety, I think it's better to let those decorator only on actual test classes, rahter than abstract class, WDYT?

Reply:
I completely agree